examples/decoding/decoding_rsa.py

```diff
@@ -0,0 +1,191 @@
+print(conditions[:10])
+
+##############################################################################
+# Let's make the event_id dictionary
+event_id = dict(zip(conditions, map(int, conds.trigger + 1)))
+event_id['0/human bodypart/human/not-face/animal/natural']
+
+##############################################################################
+# Read MEG data
+n_runs = 4  # 4 for full data (use less to speed up computations)
+fname = op.join(data_path, 'sample_subject_%i_tsss_mc.fif')
+raws = [read_raw_fif(fname % block) for block in range(n_runs)]
+raw = concatenate_raws(raws)
+
+events = mne.find_events(raw, min_duration=.002, verbose=True)
+print(events[:10])
+
+events = events[events[:, 2] <= max_trigger]
+mne.viz.plot_events(events, sfreq=raw.info['sfreq'])
+
+##############################################################################
+# Epoch data
+picks = mne.pick_types(raw.info, meg=True)
+epochs = mne.Epochs(raw, events=events, event_id=event_id, baseline=None,
+                    picks=picks, tmin=-.1, tmax=.500, preload=True)
+
+##############################################################################
+# Let's plot some conditions
+epochs['face'].average().plot()
+epochs['not-face'].average().plot()
+
+##############################################################################
+# Representational Similarity Analysis (RSA) is a neuroimaging-specific
+# appelation to refer to statistics applied to the confusion matrix
+# also referred to as the representational dissimilarity matrices (RDM).
+#
+# Compared to the approach from Cichy et al. we'll use a multiclass
+# classifier (Multinomial Logistic Regression) while the paper uses
+# all pairwise binary classification task to make the RDM.
+# Also we use here the ROC-AUC as performance metric while the
+# paper uses accuracy. Finally here for the sake of time we use
+# RSA on a window of data while Cichy et al. did it for all time
+# instants separately.
+
+# Classify using the average signal in the window 50ms to 300ms
+clf = make_pipeline(StandardScaler(),
+                    LogisticRegression(C=1, solver='lbfgs'))
+X = epochs.copy().crop(0.05, 0.3).get_data().mean(axis=2)
+y = epochs.events[:, 2]
+
+n_folds = 5
+classes = set(y)
+cv = StratifiedKFold(n_folds, random_state=0, shuffle=True)
+
+# Compute confusion matrix for each cross-validation fold
+y_pred = np.zeros((len(y), len(classes)))
+for train, test in cv.split(X, y):
+    # Fit
+    clf.fit(X[train], y[train])
+    # Predict
+    y_pred[test] = clf.predict_proba(X[test])
+
+print(y_pred[:2])
+
+
+##############################################################################
+# Compute confusion matrix using ROC-AUC
+cm = np.zeros((len(classes), len(classes)))
+for ii, train_class in enumerate(classes):
+    for jj in range(ii, len(classes)):
+        cm[ii, jj] = roc_auc_score(y == train_class, y_pred[:, jj])
+        cm[jj, ii] = cm[ii, jj]
+
+# Format class names for centered plotting
```

Comment:
oh dear this is complicated for an example. How about a manual:
`labels = [[''] * 6 + ['face'] + [''] * 12 + ['body'] + [''] * 6]`
ax.set_ytickslabel(labels)